radix-engine/src/object_modules/metadata/package.rs

```diff
@@ -262,8 +261,9 @@ impl MetadataNativePackage {
         Ok(Own(node_id))
     }
 
+    /// This method assumes that the data has been pre-checked.
```

Comment:
Can we perhaps make it not `pub`?

Perhaps we should make it not `pub` and rename it to `init_system_struct_from_pre_validated_data` or something?

And then we can re-expose some higher-level `pub` method which validates and then inits?

Reply:
Will make it non-public.